docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -193,12 +193,20 @@ test('Drag test', async t => {
 
 ## Upload Files
 
-Actions that allow you to interact with file upload input elements.
+TestCafe can't test native browser dialogs, including those during files selection. This section describes how to prevent these dialogs and select files for upload.
```

Comment:
I propose the describe the differences between `native` file upload and file upload with TestCafe.

```js
Customer upload files in the following ways:
* click on the 'Choose File' button that opens the native 'Choose file' dialog
* select the files
* initiate the upload 

Since TestCafe cannot access to the native browser dialog, it uses a custom way to manipulate with a list of uploaded files. You need to locate the 'input type=file' element that contains the list files to upload. In the case of custom upload control, you may use browser dev tools. Construct the selector points to 'input type=upload' element. 

After that call the `setFileToUpload` with a selector to the target input element and list files to upload as arguments.

Note, you don't need to call 'click' action to open the native 'Choose file' dialog or handle it with 'setNativeDialogHandler'. TestCafe set list files to upload without it.

The method `clearFileToUpload` clean up the target input in case if you are not necessary to upload file during test execution.

If you specify the wrong element for 'setFilesToUpload', 'cleanFilesToUpload' action the 'The specified selector does not match a file input element' error message will show.  In case you need to ensure that you specify the correct selector to the input type=upload
```

Reply:
We've discussed this with Vasily and decided that we do need to distinguish between the usual way and TestCafe way of uploading the files. A verbose explanation is necessary, would be too long for the `docs` section. 

I want to describe it in Recipes->Basics and see how it plays out.

Reply:
I've added this to the new recipe